### PR TITLE
Streamline CONFIG_RT_GROUP_SCHED

### PR DESCRIPTION
**Pull request by pvizeli**
Fix: #2168 

With cgroup2, we can not handle realtime schedule for now. Once we can, we can undo that change, but we should still have it global and not per board to make the same experience overall

### CODE REVIEW

File: buildroot-external/kernel/docker.config [hunk 1]
```diff
@@ -4,6 +4,8 @@ CONFIG_CFS_BANDWIDTH=y
 CONFIG_FAIR_GROUP_SCHED=y
 CONFIG_NET_SCHED=y
 
+CONFIG_RT_GROUP_SCHED=n
+
```
Comment (agners) [inline buildroot-external/kernel/docker.config:8]: That doesn't work with Kconfig. `# CONFIG_RT_GROUP_SCHED is not set` should though.

**Review by agners — CHANGES_REQUESTED** [review]

**Comment by agners** [comment]
Yeah this config got removed in #1329, and should be removed for those two boards too. Good catch! Hm, we probably should use minimal defconfigs for those two boards as well :thinking:

File: buildroot-external/kernel/docker.config (hunk 1 above)
Comment (pvizeli) [reply buildroot-external/kernel/docker.config:8]: good point :D

**Comment by pvizeli** [comment]
> Hm, we probably should use minimal defconfigs for those two boards as well 🤔

Maybe, but that would be more somethings for REL-10 I guess or at least in a new PR

**Pull request merged**